manifests/charts/base/values.yaml

```diff
@@ -9,4 +9,6 @@ global:
   istioNamespace: istio-system
 
   istiod:
-    enableAnalysis: false
\ No newline at end of file
+    enableAnalysis: false
+
+enableCRDTemplates: false
```

Comment:
Can you put a comment about why you would use this (ie helm2)

Reply:
yes, will add